packages/automerge-repo-demo-todo/package.json

```diff
@@ -13,7 +13,7 @@
     "@automerge/automerge-repo-network-broadcastchannel": "^0.1.4",
     "@automerge/automerge-repo-network-websocket": "^0.1.4",
     "@automerge/automerge-repo-react-hooks": "^0.1.4",
-    "@automerge/automerge-repo-storage-localforage": "^0.1.4",
```

Comment:
any chance the localforage adapter can be archived somewhere? for learning purposes

Reply:
sort of -- i don't know if we can implement the range operators but we could look into it